src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -429,10 +429,11 @@ private Map<String, CamundaVariable> performGetVariablesAction(String id) {
         try {
             return camundaServiceApi.getVariables(authTokenGenerator.generate(), id);
         } catch (FeignException ex) {
+            log.error("There was a problem fetching the variables for task with id: {} \n{}", id, ex.getMessage());
             throw new ResourceNotFoundException(String.format(
                 "There was a problem fetching the variables for task with id: %s",
                 id
-            ), ex);
```

Comment:
Don't you want to keep the original exception details?  Also you are catching a very generic exception here so how do we know that it was a 404?  Could possibly be a 5xx / 4xx error?

Error log is prob ok but ResourceNotFoundException assumes a 404?